### ISSUE TITLE
[SORMAS2SurvNet] Add button to send event of type cluster as outbreak to SurvNet [2]

### ISSUE THREAD

**Issue by MartinWahnschaffe** [hzi-braunschweig/SORMAS-Project#4082]
### Situation Description
Currently only cases can be send to SurvNet (see SurvnetGatewayFacade).
Recently events have been extended to also support subordinate events and to include a few important fields that were missing.

### Feature Description
Prerequisite: Extend the external survnet converter with a rest method to receive events from SORMAS

- [ ] Implement the backend functionality to push events to SurvNet (see SurvnetGatewayFacade). To do this extend the ExportParameter class with `List<String> eventUuids` (so it can later be also used for bulk sending). The receiving side is currently being implemented by the SurvNet converter team.
- [ ] Include the uuids of all cases that are related to the Event. The information is provided by the `resultingCase `field of the EventParticipant class
- [ ] Add a send button to the EventDataView, as we have in the CaseDataView (see usage of SurvnetGateway class)

### Additional Information
Implement this in a branch based on the 1.54 version, not 1.55 - it's possible that we have to release this as a hotfix.

### out of scope:
send multiple events from events directory

**Comment by DavidBaldsiefen** [comment]
I think the `de-public` label is wrong here, it is meant only for issues which are of public interest, and which will then be automatically fetched and displayed on the sormas-oegd website. The issuetracker is not live yet though, so it would actually nice if you kept the label for now so I can use it to test my code 🙂

**Comment by bernardsilenou** [comment]
@MartinWahnschaffe @kwa20  Just a comment which i do not know if it applies here or there is another issue for it:
Since in survnet, a case can result from only one event among the set of all possible events, we may need an extra cleaning step of assigning the resulting case to only one event before pushing.
For a resulting case X with source event E1 (date of event Jan 1) and E2 (date of event Jan 5),  We thought of assigning the case to E1, ie earliest in term of date of event.

In future refinement stage, we may latter need a method to compute the most likely source event by comparinig the date of event and onset of case.

**Comment by MartinWahnschaffe** [comment]
> 
> 
> I think the `de-public` label is wrong here, it is meant only for issues which are of public interest, and which will then be automatically fetched and displayed on the sormas-oegd website. The issuetracker is not live yet though, so it would actually nice if you kept the label for now so I can use it to test my code 🙂

Hm, from my understanding this is quite important for the Gesundheitsämter. If there is a need to distinguish towards "public" interest, we need to clearly define where this label applies.

**Comment by DavidBaldsiefen** [comment]
> Hm, from my understanding this is quite important for the Gesundheitsämter. If there is a need to distinguish towards "public" interest, we need to clearly define where this label applies.

Sure I agree, it definetely needs to be clarified. My last understanding was that the folks over at HZI want to decide what gets this label and what doesn't. But there is not final decision on that yet as the system is still in development anyway.

**Comment by MartinWahnschaffe** [comment]
@markusmann-vg Please clarify whether we also need to include the subordinate events.

**Comment by markusmann-vg** [comment]
@MartinWahnschaffe The complete hierarchy from the event should be included (Superordinate & Subordinate events)

**Comment by AndyBakcsy-she** [comment]
verified ticket on the latest version of 1.56.0 on test1402.sormas-oegd.de
